deployments/README.md

```diff
@@ -0,0 +1,72 @@
+# Nanocloud deployments
+
+This directory is a collection of tools to deploy Nanocloud.
+
+## Install with ansible
+
+### Dependencies
+
+You need ansible 1.9 or above to use this installation
+
+Install dependencies
+
+```
+ansible-galaxy install franklinkim.docker
+ansible-galaxy install franklinkim.docker-compose
+```
+
+> Note: You may need to have root privilege to install those dependencies. You
+> won't need it to run deployment.
+
+### Configure
+
+Configure where *ansible* will deploy your installation by modifying
+*ansible_hosts* file.
+
+```
+[nanocloud]
+api ansible_host=127.0.0.1
+```
+
+In this file, you can specify several server and give name to them to deploy
+multiple instance of nanocoud:
+
+```
+[nanocloud]
+api-instance1 ansible_host=10.0.0.2
+api-customerX ansible_host=10.0.0.3
+```
+
+You can change hosts username to use un *nanocloud.yml*.
+
+Use the file *deployments/roles/nanocloud/files/nanocloud/config.env* to
+override configuration variable like you should normally do with *config.env*
+file
+
+### Run
+
+Run Playbook
+
+```
+ansible-playbook nanocloud.yml
+```
+
+## Run multiple instance on a same host
+
+Script *deploy-pr.sh* is here to help you deploy a specific Github Pull Request
+on any environement without closing another instance.
```

Comment:
on any environement -> on any environment. 
Script -> The script
without closing another instance -> in parallel